src/Widgets/ImageWidget/DimensionsPicker.tsx

```diff
@@ -14,7 +14,60 @@ export const DimensionsPicker = connect(function DimensionsPicker({
     <div
       className={`dimensions-picker scrivito_detail_content scrivito_${theme}`}
     >
-      Dimensions picker
+      <div className="row">
+        <div className="col-6">
+          <div className="scrivito_detail_label">
+            <span>Width</span>
+          </div>
+
+          <div className="item_content">
+            <div className="input_group">
```

Comment:
Could you add the nice "full box" highlight style as dartagnan is using?

![Screenshot 2024-12-13 at 11 58 15](https://github.com/user-attachments/assets/2666293d-6f67-4dcb-9334-474c44decbe2)

Currently only the input field itself is highlighted, not the select box as well:
![Screenshot 2024-12-13 at 12 01 09](https://github.com/user-attachments/assets/03e7e0f4-3bd9-40fc-b9ed-5d82b1602a1b)

Two more observations:
* The inner spacing could be improved.
* Dartagnan limits the width of the element to 100px, which looks quite nice.